.changeset/giant-boats-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/compiler': patch
```

Comment:
This will have to be a major because we are depending on runtime APIs that won't exist for older versions of Astro.

Reply:
Will it? `astro@2.10.13` has `"@astrojs/compiler": "^1.8.0"` as a dependency. `astro@3.0.0-rc.5` has `"@astrojs/compiler": "^2.0.0"` as a dependency.

The compiler version in 3.0.0-rc.5 is currently broken, so this is a bug fix for that release line.

Reply:
Ok, didn't realize we already did a major for the compiler, nm then.